views/js/previewer/adapter/item/qtiItem.js

```diff
@@ -32,27 +32,38 @@ define([
      * List of required plugins that should be loaded in order to make the previewer work properly
      * @type {Object[]}
      */
-    const defaultPlugins = [{
-        module: 'taoQtiTestPreviewer/previewer/plugins/controls/close',
-        bundle: 'taoQtiTestPreviewer/loader/qtiPreviewer.min',
-        category: 'controls'
-    }, {
-        module: 'taoQtiTestPreviewer/previewer/plugins/navigation/submit/submit',
-        bundle: 'taoQtiTestPreviewer/loader/qtiPreviewer.min',
-        category: 'navigation'
-    }, {
-        module: 'taoQtiTestPreviewer/previewer/plugins/tools/scale/scale',
-        bundle: 'taoQtiTestPreviewer/loader/qtiPreviewer.min',
-        category: 'tools'
-    }, {
-        module: 'taoQtiTest/runner/plugins/tools/itemThemeSwitcher/itemThemeSwitcher',
-        bundle: 'taoQtiTest/loader/testPlugins.min',
-        category: 'tools'
-    }, {
-        module: 'taoQtiTestPreviewer/previewer/plugins/content/enhancedReadOnlyMode',
-        bundle: 'taoQtiTestPreviewer/loader/qtiPreviewer.min',
-        category: 'content'
-    }];
+    const defaultPlugins = [
+        {
+            module: 'taoQtiTestPreviewer/previewer/plugins/controls/close',
+            bundle: 'taoQtiTestPreviewer/loader/qtiPreviewer.min',
+            category: 'controls'
+        },
+        {
+            module: 'taoQtiTestPreviewer/previewer/plugins/navigation/submit/submit',
+            bundle: 'taoQtiTestPreviewer/loader/qtiPreviewer.min',
+            category: 'navigation'
+        },
+        {
+            module: 'taoQtiTestPreviewer/previewer/plugins/tools/scale/scale',
+            bundle: 'taoQtiTestPreviewer/loader/qtiPreviewer.min',
+            category: 'tools'
+        },
+        {
+            module: 'taoQtiTest/runner/plugins/tools/itemThemeSwitcher/itemThemeSwitcher',
+            bundle: 'taoQtiTest/loader/testPlugins.min',
+            category: 'tools'
+        },
+        {
+            module: 'taoQtiTestPreviewer/previewer/plugins/content/highlighter',
```

Comment:
isn't this going to enable the plugin for all previews? We need to enable it only for manual scoring